<!-- -->

<launch>

    <node name = "I2V" pkg = "obu_parsing" type = "I2V.py" output = "screen">
    </node>

    <node name = "obu_to_xy" pkg = "obu_to_xy" type = "obu_to_xy" output = "screen">
        <param name = "sub_obu_str" type="string" value = "/I2V_info"/>
        <param name = "sub_lidar_str" type="string" value = "/obstacle_array"/>
        <param name = "pub_obu_str" type="string" value = "/msgs_obu"/>
        <param name = "pub_lidar_str" type="string" value = "/obstacle_array_obu"/>

        <param name = "o_life_time" type="int" value = "1"/>
        <param name = "o_scale_x" type="double" value = "2.0"/>
        <param name = "o_scale_y" type="double" value = "2.0"/>
        <param name = "o_scale_z" type="double" value = "2.0"/>
        <param name = "o_angle" type="double" value = "0.0"/>
        
        <param name = "distance_emergency" type="int" value = "50"/>
    </node>
</launch>

